xml/security_vpnserver.xml

```diff
@@ -1281,9 +1281,9 @@ DNSSERVERS='&dnsip;'
 EOT</screen>
     <para>
      To check whether the entry has been successfully inserted into
-     <filename>/etc/resolv.conf</filename>, execute:
+     <filename>/var/run/netconfig/resolv.conf</filename>, execute:
```

Comment:
IHMO we should stick to /etc/resolv.conf in this section, too.

Reply:
Done.